### PR TITLE
Rename properties page namespaces and classes

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Description

- [x] Rename `Views/Pages/Properties*.xaml(.cs)` with `Views/Properties/*.xaml(.cs)`
- [x] Rename `ViewModels/Properties/*Properties.cs` with `ViewModels/Properties/*.cs`

### Motivation and Context

- Related #11257
- Related #4180

### Validation

- [x] Built and ran the app
- [x] Tested the changes for accessibility

### Screenshots

Solution Explorer
image

### CODE REVIEW

**Comment by yaira2** [comment]
@onein528 can you fix the merge conflicts?

**Review by yair100 — DISMISSED** [review]
LGTM

**Comment by onein528** [comment]
Wait a sec.

**Comment by onein528** [comment]
How can I fix? VS2022 looks like no conflicts

**Comment by yaira2** [comment]
The only thing that's changed was some strings but that shouldn't be causing this kind of conflict

**Comment by onein528** [comment]
can you force-push?

**Comment by onein528** [comment]
Done.

File: src/Files.App/Views/Properties/MainPage.xaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ﻿<Page
-	x:Class="Files.App.Views.Properties"
+	x:Class="Files.App.Views.Properties.MainPage"
```
Comment (yaira2) [inline src/Files.App/Views/Properties/MainPage.xaml:2]: I don't think `MainPage` is a good, should be something unique to properties.

Comment (onein528) [reply src/Files.App/Views/Properties/MainPage.xaml:2]: MainPropertiesPage?

Comment (yaira2) [reply src/Files.App/Views/Properties/MainPage.xaml:2]: 👍

**Comment by 0x5bfa** [comment]
@yair100 Done.

**Pull request merged**